cmd/child.go

```diff
@@ -31,13 +31,16 @@ func child(_ *cobra.Command, args []string) error {
 	containerID := generateContainerID()
 
 	// Create a temporary directory to extract the image contents
-	tempDir := fmt.Sprintf("./%s-tempfs", containerID)
+	tempDir := fmt.Sprintf("/var/lib/container-runtime/%s-tempfs", containerID)
+
+	// Defer the cleanup function to remove the temp directory on exit
+	defer cleanupTempDir(tempDir)
```

Comment:
this should be run after `if err := os.MkdirAll`

Reply:
Yes, academically speaking it makes more sense, but the effect is the same